generator/execute.go

```diff
@@ -159,7 +159,11 @@ func assembleGolangFile(w io.Writer, f *File) {
 }
 
 func importsWrapper(src []byte) ([]byte, error) {
-	return imports.Process("", src, nil)
+	dir, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	return imports.Process(dir, src, nil)
```

Comment:
Process expects a file name to do path-relative resolution (e.g. nested vendor/).  How hard would it be to plumb that down?

@lavalamp @sttts

Reply:
Compare https://github.com/kubernetes/gengo/pull/115. Answer: simple.